### PR TITLE
fix(aws): Flagging elb security group autocreation (defaults to off)

### PR DESCRIPTION
**Pull request by jeyrschabu**
- made auto creation of <appname>-elb group for new ELBs configurable
- defaulting to off

### CODE REVIEW

**Comment by jeyrschabu** [comment]
@cfieber PTAL (cc: @ewiseblatt )

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy [hunk 1]
```diff
@@ -71,6 +73,9 @@ class UpsertAmazonLoadBalancerAtomicOperation implements AtomicOperation<UpsertA
   @Autowired
   SecurityGroupLookupFactory securityGroupLookupFactory
 
+  @Value("\${aws.autoCreateELBGroup:false}")
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy]: could you add this flag to the AwsConfiguration.DeployDefaults class then autowire that bean in -- that is where the flag for the application security group creation is (the config namespace would then be `aws.defaults.`)

**Comment by jeyrschabu** [comment]
@ewiseblatt this will take care of that. the autocreation creates a <appname>-elb group and a <appname> group if it doesnt exist.

**Comment by ewiseblatt** [comment]
ok. I assume this is good since cameron is happy. If not I'll ping you
again.

**Pull request merged**